lang/doc/v4/funcs/internal-functions.hjson

```diff
@@ -5,6 +5,18 @@
       params: [ "T", "String" ]
       doc: "Check value type"
       paramsDoc: [ "The value", "The type name" ]
+    },
+    {
+      name: "Issue"
+      params: [ "String", "String", "Int", "Int", "Boolean", "Script|Unit", "Int" ]
+      doc: "Issue action detailed constructor"
+      paramsDoc: [ "Token name", "Token description", "Amount of the token", "Number of digits in decimal part", "Reissue ability flag", "Smart asset script (optional)", "Sequential number"]
+    },
+    {
+      name: "Issue"
```

Comment:
are they will be shown in the surfboard help, for example?

Reply:
should be